config/auth.php

```diff
@@ -102,6 +102,8 @@
     'password_reset_lifetime' => env('AUTH_PASSWORD_RESET_LIFETIME', 1800),
     'password_min_length' => env('AUTH_PASSWORD_MIN_LENGTH', 8),
     'password_max_length' => env('AUTH_PASSWORD_MAX_LENGTH', 30),
+    'password_shape_pattern' => env('AUTH_PASSWORD_SHAPE_PATTERN', '^(?=.*[a-z])(?=.*[!@#$%^&*])'),
```

Comment:
add
AUTH_PASSWORD_SHAPE_PATTERN
AUTH_PASSWORD_SHAPE_WARNING
to
https://github.com/OpenStackweb/openstackid/blob/main/.env.example